include/gba/defines.h

```diff
@@ -9,10 +9,10 @@
 #define IWRAM_DATA __attribute__((section("iwram_data")))
 #define EWRAM_DATA __attribute__((section("ewram_data")))
 
+// TODO: Move these elsewhere?
```

Comment:
Where were you thinking?

Reply:
Maybe global.h?

It's just that they're GCC-specific (the "section" attribute above is as well, technically), so having them in a platform header doesn't sound right to me.